packages/sveltekit/src/createClient.ts

```diff
@@ -11,7 +10,7 @@ import type { TypedSupabaseClient } from './types';
 export function createClient(
   supabaseUrl: string,
   supabaseKey: string,
-  options?: SupabaseClientOptionsWithoutAuth<App.Supabase['SchemaName']>,
```

Comment:
Actually, I'd say the sveltekit option implementation is exactly what we want to use across the board. Sorry if that wasn't clear.

Reply:
It should be the same for all, the SupabaseClientOptionsWithoutAuth type was unnecessary as the type `SupabaseClientOptions` exported from the shared package omits the auth key
https://github.com/supabase/auth-helpers/blob/7ddaa6a1dbb9262ea15620d661b6d1691a23623f/packages/shared/src/types.ts#L9

Reply:
ah yes, perfect, I missed that it was coming from shared, sweet 👍

Reply:
Maybe we should also rename the type to reduce confusion

Reply:
Yah, I think that might be good